diff --git a/dist/lib/utils-common/jsx.js b/dist/lib/utils-common/jsx.js
index 1e611039dbf2cfd85ce6dbf7f33be5bd80c3589e..e2d99881d882bf1d8d6541f5b21082fcf792dc89 100644
--- a/dist/lib/utils-common/jsx.js
+++ b/dist/lib/utils-common/jsx.js
@@ -75,7 +75,7 @@ export function renderElement(element) {
         return "";
     }
     const { tag, props, children } = element;
-    let html = "";
+    let html = "\n";
     if (typeof tag === "function") {
         if (tag === Raw) {
             return String(props.html);
@@ -86,9 +86,6 @@ export function renderElement(element) {
         }
         return renderElement(tag(Object.assign({ children }, props)));
     }
-    if (blockElements.has(tag) && renderPretty && html) {
-        html += "\n";
-    }
     html += "<";
     html += tag;
     for (const [key, val] of Object.entries(props ?? {})) {
